### ISSUE TITLE
Infinit loading word - solo

### ISSUE THREAD

**Issue by AntonioVillazon** [TheBritishBrotherhood/ArdentCraft-The-Beginning#170]
Hi, I juste try the last version 4.3, but at the creation the world don't stop loading. I was so happy to see a new update, but don't work... I have try withe the 4.2.02 and the world load properly.
Someone have same problem ?

**Comment by ladyannie68** [comment]
Yes I can't load in since the latest update :( been so looking forward to seeing if things were fixed.

**Comment by suduko666** [comment]
im also having this issue ive tried reinstalling the pack repairing minecraft option and trying to create multiple worlds on each all ended in infinite loading world screen

**Comment by mastertrain501** [comment]
Ya this is definitely an issue for the 4.3 version of the modpack. New users, if you use the 4.2.02 version (previous version) of the modpack, it should fix any world loading issues.

Just hope this helps anyone using the modpack, since support seems to be dead...
